### DISCUSSION TITLE
HTTP Endpoint parameters and query

### DISCUSSION THREAD

**Discussion by ukepogluradr** [elsa-workflows/elsa-core#1474] [Q&A] [answered]
hi,

How can I pass a parameter or a query string to a HTTP Endpoint workflow?  Lets assume that, I will create  parent/child workflows.
The Parent child has a HTTP Endpoint  activity. It gets a parameter or query string over http request and then pass it to the child workflow. The child worklfow has a Http response activity. It shows the parameter to the screen as a html output.

thanks in advance,
umit

**Comment by ukepogluradr** [comment, marked as answer]
hi,

I've found a solution for this.  I've created a workflow. First activity is a HTTP Request  and second activity is a HTTP response.
I've written this code snippet as JavaScript in the Content area:

var adi = queryString("personname");
`Hello ${adi}`

and then I've published this workflow. For run, I've used http://localhost:5000/show?personname=John. That's it.


Thanks,
umit.

**Reply by swuecho** [reply]
https://elsa-workflows.github.io/elsa-core/docs/next/expressions/expressions-javascript#querystring
